MAUI/Maps/tooltip.md

```diff
@@ -271,6 +271,110 @@ The [`ShowMarkerTooltip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Ma
 
 ## Appearance customization
 
+You can customize the below appearances of the tooltip.
+
+* **Background** - Change the background color of the tooltip in the maps using the `MapTooltipSettings.Background` property.
+* **Padding** - Change the padding of tooltip text in the maps using the `MaptooltipSettings.Padding` property.
+* **TextStyle** - Change the tooltip text appearance such as `TextColor`, `FontSize`, `FontAttributes` and `FontFamily` in the maps using `MapTooltipSettings.TextStyle` property.
```

Comment:
* **TextStyle** - Change the tooltip text appearance such as `TextColor,` `FontSize,` `FontAttributes,` and `FontFamily` in the maps using the `MapTooltipSettings.TextStyle` property.

Reply:
I have changed.